Doc/deprecations/pending-removal-in-future.rst

```diff
@@ -11,7 +11,6 @@ although there is currently no date scheduled for their removal.
 
 * :mod:`builtins`:
 
-  * ``~bool``, bitwise inversion on bool.
```

Comment:
If this gets merged, I would open a new PR to move this note to the 3.14 pending removals & backport that.